### ISSUE TITLE
Add Features & Analysis to Secondary Column

### ISSUE THREAD

**Issue by simonsinclair** [bbc/simorgh#5701]
### Is your feature request related to a problem? Please describe.
Add Features & Analysis to the Secondary Column, following the merge of https://github.com/bbc/simorgh/pull/5417. Features & Analysis will be rendered using a similar pattern to CPS Related Content, where `content` below is Features & Analysis data:

```jsx
<StoryPromoUl>
  {content
    .map(item => (
      <StoryPromoLi key={item.id || item.uri}>
        <StoryPromo item={item} />
      </StoryPromoLi>
    ))}
</StoryPromoUl>;

```

### Describe the solution you'd like
1. Create a new FeaturesAnalysis container. 
2. The Features and Analysis end-point is the front page. The data comes from the CPS group 'Features', which is held on the front page.
E.g. for the Nepali service, https://ares-broker.api.bbci.co.uk/api/index/nepali/front_page
```
content: {
groups: [
type: "feature-main",
title: "Features",
```
This data will need to pass the relevant properties to a `StoryPromo` component as well as to a section label.

3. Use `psammead-story-promo-list` in a similar way shown in the example above to render the dummy fixture data in a secondary column component container.
4. Use `psammead-section-label` to create a "Features & Analysis" section label from the fixture value created in step 2.
5. A11y - This should be wrapped in a section html element with a role region attribute which is labelled via an aria-labelledby attribute which maps to the h2. Below is an example from a front page section

<img width="628" alt="Screen Shot 2020-03-02 at 11 56 12" src="https://user-images.githubusercontent.com/1061401/75674524-22a30d80-5c7d-11ea-94f6-a0c1a8edd471.png">

6. Snapshot tests.

### Describe alternatives you've considered
n/a

### Testing notes
Given I use a screen reader
When I navigate to the features and analysis section on an STY page
Then a region landmark must be announced e.g. Region
And this must be announced with a label 
And the label must be the same as the h2 (e.g. Features and analysis) and be unique to the page
And this should not be inside the main landmark or any other landmark

Given I am using a screenreader
When I navigate to a features and analysis component promo
And there is no content within the promo below the link
Then the promo should not contain heading semantics

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

### Additional context
- https://paper.dropbox.com/doc/Simorgh-WS-STY-onward-journeys--AvIEmS7B0c8VTK8ym0qb6uTrAg-gSSePsYXjWttnHdEIwWYI
 - https://www.bbc.com/thai/thailand-51736465
- The component header is also taken from the front page data. See #5794. The CPS group has a semantic group name with a translation. E.g. for Nepali (https://ares-broker.api.bbci.co.uk/api/index/nepali/front_page) the CPS group for Features and Analysis is 'Features'. This group has a group.strapline.name of "अन‌ि यो पनि", so the heading for the Features and Analysis onward journey on the STY page is "अन‌ि यो पनि".
```
strapline: {
name: "अन‌ि यो पनि"
},
semanticGroupName: "Features"
```

**Comment by simonsinclair** [comment]
Unblocked following the merge of https://github.com/bbc/simorgh/pull/5417.

**Comment by mykeels** [comment]
Is it a safe bet that the features&analysis data will look like page promos?

E.g.

https://github.com/bbc/simorgh/blob/latest/data/pidgin/cpsAssets/tori-49450859.json#L355-L382

**Comment by simonsinclair** [comment]
> Is it a safe bet that the features&analysis data will look like page promos?

I don't think it's safe to bet on anything at this stage. We know we will have data for the relevant fields (e.g. headline, image source, etc) as shown in the designs. This issue recommends a temporary fixture be used, which could be as simple as the following:

```json
[
  {
    "headline": "",
    "imgSrc": "",
    "href": ""
  },
  {
    "headline": "",
    "imgSrc": "",
    "href": ""
  }
]
```

**Comment by mykeels** [comment]
If we make the fixture data structure so simple, it can no longer be rendered by the `StoryPromo` component, which expects its `item` prop to look like the promo objects in the page data.

```json
{
  "headlines": {
    "shortHeadline": "Meet boys who dey convert cassava to electricity",
    "headline": "Meet boys who dey convert cassava to electricity"
  },
  "locators": {
    "assetUri": "/pidgin/44508901",
    "cpsUrn": "urn:bbc:content:assetUri:/pidgin/44508901"
  },
  "summary": "James den Kwesi Ansah want convert most of Ghana's cassava waste into cheap electricity supply for schools, hospitals den villages.",
  "timestamp": 1529397290,
  "cpsType": "MAP",
  "indexImage": {
    "href": "http://c.files.bbci.co.uk/C576/production/_102105505_p06bhwsd.jpg",
    "path": "/cpsprodpb/C576/production/_102105505_p06bhwsd.jpg",
    ...
    "type": "image"
  },
  "id": "urn:bbc:ares::asset:pidgin/44508901",
  "type": "cps"
}
```

**Comment by mykeels** [comment]
The issue description says to use `<StoryPromo item={item} />`. Is this no longer necessary?

**Comment by simonsinclair** [comment]
Hey Michael, hopefully everything was clarified over the call. 👍

**Comment by mykeels** [comment]
Following @greenc05's suggestions in https://github.com/bbc/simorgh/pull/5751, it is clear that 

- We need to override the `psammead-section-label`'s styles
- Re-using the StoryPromo component as is, creates a11y concerns, such as:
  - using DIV instead of H3 in feature title

**Comment by mykeels** [comment]
How do we add translations for the column title?

**Comment by mykeels** [comment]
Create a separate issue to modify Related Content and Features & Accessibility, to use a DIV for their titles, rather than H3 tags. ✅https://github.com/bbc/simorgh/issues/5793

See https://www.test.bbc.com/pidgin/23259748, and inspect Related Content titles. (Provide Screenshots)

**Comment by mykeels** [comment]
Create separate issue to get translated headings for Features & Analysis, Top Stories components. ✅https://github.com/bbc/simorgh/issues/5794

**Comment by mykeels** [comment]
Create separate issue finding a DRY approach for CPS Related Content, Features & Analysis containers, to be implemented when working on Top Stories container. ✅https://github.com/bbc/simorgh/issues/5795

See line https://github.com/bbc/simorgh/pull/5751/files#diff-f93c9baefa363f1d9525e83dba7e6627R19-R34 where the Wrapper and StyledSectionLabel are repeated across the containers.

**Comment by lizcameron** [comment]
Issue updated with information on the data endpoint for the Features and Analysis onward journey. See point 1. in the issue description.
